docs/reviewing.md

```diff
@@ -0,0 +1,36 @@
+Two output checkers independently review each file and record their decisions using the buttons within Airlock.
+Whilst viewing a file in an Airlock release request, they have the option of: 
+
+* **Approve** — output meets disclosure requirements and is safe to be released
+* **Reject** — output is not currently acceptable for release. 
+
+!!!warning "Retrictions for authors"
+    Airlock does not permit users to approve or reject files that are part of release requests they created themselves.
+
+
+## Viewing release requests on Airlock
+
+1. Output checkers navigate to the Requests list to view all open requests
+    awaiting review. Note that if an output checker is the author of a request, they
+    will not see it in the outstanding requests list. 
+1. Output checkers identify the request by workspace name and requester username.
+1. The files included in the request can be viewed in the file
+    browser within Airlock. If required (e.g. in order to perform calculations
+    in a spreadsheet), files can also be downloaded from Airlock. Please note
```

Comment:
```suggestion
    in a spreadsheet), output checkers can also download files from Airlock. Please note
```

Reply:
Just to make it a bit clearer this is only for output checking, not for viewing outputs

Reply:
I've rewritten this last point in the same style as the previous ones, and added a bit more clarification about what the download option is intended for